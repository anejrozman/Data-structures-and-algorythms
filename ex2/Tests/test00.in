5,20,0
29.78,41.97,47.78,58.97
9.19,13.78,18.189999999999998,21.78
84,46.5,100,74.5
65.45,8.86,83.45,34.86
45.76,11.56,57.76,25.560000000000002

29.78,41.97
29.78,58.97
47.78,41.97
47.78,58.97
9.19,13.78
9.19,21.78
18.189999999999998,13.78
18.189999999999998,21.78
84,46.5
84,74.5
100,46.5
100,74.5
65.45,8.86
65.45,34.86
83.45,8.86
83.45,34.86
45.76,11.56
45.76,25.560000000000002
57.76,11.56
57.76,25.560000000000002
