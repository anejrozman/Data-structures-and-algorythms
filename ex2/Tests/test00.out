148.54336854690155
147.47721563402757
141.75558409702836
147.09964826066482
144.49866707174516
145.25871215540374
141.75558409702836
144.70880258929742
154.11654459068993
142.46683640471508
184.75249655773405
153.86284995773434
165.91660511273295
148.06005943572455
179.6653117275668
161.9382250700225
153.3023600152252
144.51939245691386
157.31129925712943
150.04931812551106
